### PR TITLE
Patiente parte 2

### PR DESCRIPTION
**Pull request by lomky**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Abortion tab translation
* call log tab
* change log tab
  * Not in this ticket: history tracks translation handling

screenshot from 2019-02-16 21-04-19

screenshot from 2019-02-17 08-56-39

screenshot from 2019-02-17 09-11-17

It relates to the following issue #s: 
* Bumps #1572 
  * Finishes it if #1608 is already merged!

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -74,6 +74,23 @@ en:
       sign_in: Sign In
       sign_out: Sign Out
   patient:
+    abortion:
```
Comment (colinxfleming) [inline config/locales/en.yml]: thoughts about changing this to `abortion_information` so it matches the partial? or is that overkill?

File: config/locales/en.yml [hunk 2]
```diff
@@ -74,6 +74,23 @@ en:
...
+      clinic_section:
+        clinic: Clinic
+        medicaid_only_toggle: Enable only Medicaid clinics
+        naf_only_toggle: Enable only NAF clinics
+        referred_to_clinic: Referred to clinic
+        resolved_without_fund: Resolved without assistance from %{fund}
+        title: Clinic details
+        ultrasound_completed: Ultrasound completed?
```
Comment (colinxfleming) [inline config/locales/en.yml]: :chef's kiss:

**Comment by lomky** [comment]
Actually ready for review now @colinxfleming! :laughing:

**Comment by lomky** [comment]
Ready for Spanish review! @montanezp

File: config/locales/en.yml [hunk 3]
```diff
@@ -74,6 +79,39 @@ en:
...
+      cost_section:
+        NAF_pledge: National Abortion Federation pledge
```
Comment (colinxfleming) [inline config/locales/en.yml]: might not matter but should be downcase this for consistency?

Comment (lomky) [inline config/locales/en.yml]:
```suggestion
        naf_pledge: National Abortion Federation pledge
```

File: config/locales/es.yml [hunk 4]
```diff
@@ -74,6 +79,39 @@ es:
...
+      clinic_section:
+        clinic: Clínica
+        medicaid_only_toggle: Habilitar solo las clínicas de Medicaid
+        naf_only_toggle: Habilitar solo clínicas de la NAF
+        referred_to_clinic: Referido a la clínica
+        resolved_without_fund: Resuelto sin ayuda de %{fund}
+        title: Detalles de la clínica
+        ultrasound_completed: "¿Ultrasonido completado?"
+      cost_section:
+        NAF_pledge: Promesa de la Federación Nacional de Aborto
```
Comment (lomky) [inline config/locales/es.yml]:
```suggestion
        naf_pledge: Promesa de la Federación Nacional de Aborto
```

File: app/views/patients/_abortion_information.html.erb [hunk 5]
```diff
@@ -56,26 +58,26 @@
...
             </div>
           </div>
 
           <div class="col-sm-6">
             <div id="pledge-contributions">
-              <%= f.number_field :patient_contribution, label: 'Patient contribution', autocomplete: 'off', prepend: '$' %>
-              <%= f.number_field :naf_pledge, label: 'National Abortion Federation pledge', autocomplete: 'off', prepend: '$' %>
-              <%= f.number_field :fund_pledge, label: "#{FUND} pledge", autocomplete: 'off', prepend: '$', label_class: 'tooltip-header-input', data: { 'tooltip-text' => pledge_limit_help_text_options.join("<br />") } %>
+              <%= f.number_field :patient_contribution, label: t('patient.abortion_information.cost_section.patient_contribution'), autocomplete: 'off', prepend: '$' %>
+              <%= f.number_field :naf_pledge, label: t('patient.abortion_information.cost_section.NAF_pledge'), autocomplete: 'off', prepend: '$' %>
```
Comment (lomky) [inline app/views/patients/_abortion_information.html.erb]:
```suggestion
              <%= f.number_field :naf_pledge, label: t('patient.abortion_information.cost_section.naf_pledge'), autocomplete: 'off', prepend: '$' %>
```

File: config/locales/es.yml [hunk 6]
```diff
@@ -74,6 +79,39 @@ es:
...
+        fund_pledge: promesa de %{fund}
+        naf_pledge: Promesa de la Federación Nacional de Aborto
+        outstanding_balance: Saldo pendiente
+        patient_contribution: Contribución del paciente
+        title: Detalles del costo
+      title: Informacion de aborto
+    call_log:
+      record_new_call: Grabar nueva llamada
+      table:
+        actions: COMPORTAMIENTO
```
Comment (montanezp) [inline config/locales/es.yml]: acciones

**Pull request merged**